pretixprint/app/src/main/java/eu/pretix/pretixprint/print/ESCPOSRenderer.kt

```diff
@@ -90,18 +92,27 @@ class ESCPOSRenderer(private val receipt: JSONObject, private val charsPerLine:
             Drawer1(0),
             Drawer2(1)
         }
-    }
 
+        enum class Dialect(val description: String) {
+            EpsonDefault("Epson, Bixolon, Metapace, SNBC"),
+            Sunmi("Sunmi"),
+        }
+    }
 
     fun render(): ByteArray {
         out.clear()
         init()
-        characterCodeTable(CharacterCodeTable.WPC1252.codeTable)
-        internationalCharacterSet(InternationalCharacterSet.Germany.country)
+        if (dialect == Dialect.EpsonDefault) {
+            characterCodeTable(CharacterCodeTable.UserDefined2.codeTable)
```

Comment:
⚠️ Character Code Table changes with this PR from `WPC1252` to `UserDefined2` for epson dialect.

Reply:
Yup, was not intended